ext/gosu/gosu.i

```diff
@@ -658,6 +658,32 @@ namespace Gosu
         std::unique_ptr<Gosu::ImageData> image_data = $self->data().subimage(x, y, w, h);
         return image_data.get() ? new Gosu::Image(std::move(image_data)) : nullptr;
     }
+
+    %newobject from_blob;
+    static Gosu::Image* from_blob(const std::string& blob, int width, int height)
```

Comment:
If `Gosu::Image` works (because it's default constructible now?), we should always just return images directly instead of working with pointers and `%newobject`.